tests/e2e/setup/chain.go

```diff
@@ -23,7 +23,7 @@ const (
 
 var (
 	encodingConfig sdkparams.EncodingConfig
-	cdc            codec.Codec
+	Cdc            codec.Codec
```

Comment:
why do you create this as a global? I think it should be part of the `E2E` suite struct.

Reply:
move the Cdc into E2E test suite